src/components/Text/src/Text.vue

```diff
@@ -37,6 +37,15 @@ export default {
 			default: undefined,
 			validator: (size) => size >= MIN_SIZE && size <= MAX_SIZE,
 		},
+		/**
+		 * Variant, allows four custom font styles through the Theme component
+		 * @values body, title, headline, label
+		 */
+		variant: {
+			type: String,
+			default: 'body',
+			validator: (variant) => ['body', 'title', 'headline', 'label'].includes(variant),
```

Comment:
this should be `paragraph` instead of `body`

Reply:
`paragraph` is what Website uses and we should stay aligned with Website where possible given they're our only and biggest user

Reply:
I'd really prefer to keep somewhat agnostic to UI in one product. It used to be body, now it's paragraph, next week it could be text -- the point is Website should feel free to change their label and output however they see fit. We should come up with something that doesn't need to change. The primary goal of `body` is this is the default styles applied to the entire body of the document -- this font and weight is used for more than just paragraphs so it's a more inclusive and descriptive name of how it will be applied in this and every other component.

Reply:
I realized that the variant `title` was following an old name and didn't conform to our existing internal data, the theme spec, or the new name. So updated `title` to `heading`